source/draw/RasterBgr.ooc

```diff
@@ -117,7 +117,7 @@ RasterBgr: class extends RasterPacked {
 		x, y, imageComponents: Int
 		requiredComponents := 3
 		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntSize2D new(x, y))
+		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents, true), IntSize2D new(x, y))
```

Comment:
I think it was this one : https://msdn.microsoft.com/en-us/library/windows/desktop/aa372603%28v=vs.85%29.aspx
`PdhEnumObjectsH`, we should make our API more like this. And remember to add `1` to the result, but only on Windows XP :D